docs/dev_guide/contents/extending_fido.rst

```diff
@@ -140,13 +154,17 @@ To make these complex queries easily processable by the clients the ``AttrWalker
 It does this by converting the input query to a set of queries which are ORed, but are complete queries.
 This means the list of queries is an **OR** of **ANDs** (technically called `disjuntive normal form <https://en.wikipedia.org/wiki/Disjunctive_normal_form>`__).
 
-Each query in the list of ORs contains all the information about that query so for example if the user provided a query like::
+Each query in the list of ORs contains all the information about that query so for example if the user provided a query like
 
-  a.Time("2020/02/02", "2020/02/03") & (a.Instrument("AIA") | a.Instrument("HMI"))
+.. code-block:: python
+
+    a.Time("2020/02/02", "2020/02/03") & (a.Instrument("AIA") | a.Instrument("HMI"))
 
 it would be passed to the client as::
 
-  (a.Time("2020/02/02", "2020/02/03") & a.Instrument("HMI")) | (a.Time("2020/02/02", "2020/02/03") & a.Instrument("AIA"))
+.. code-block:: python
+
+    (a.Time("2020/02/02", "2020/02/03") & a.Instrument("HMI")) | (a.Time("2020/02/02", "2020/02/03") & a.Instrument("AIA"))
```

Comment:
This code block isn't rendering properly.

Reply:
There's a `::` at the end of the previous line so it's all being interpreted as a code block.